pallets/transaction-multi-payment/benchmarking/src/mock.rs

```diff
@@ -163,11 +163,9 @@ impl AssetPairAccountIdFor<AssetId, u64> for AssetPairAccountIdTest {
 		let mut a = asset_a as u128;
 		let mut b = asset_b as u128;
 		if a > b {
-			let tmp = a;
-			a = b;
-			b = tmp;
+			std::mem::swap(&mut a, &mut b)
```

Comment:
fancy

Reply:
...standard. I would say it is fancy to perform swap on your own.